src/views/forms/TradeForm/TradeSizeInputs.tsx

```diff
@@ -106,15 +109,30 @@ export const TradeSizeInputs = () => {
   };
 
   const inputToggleButton = (
-    <$ToggleButton
-      isPressed={showUSDCInputOnTablet}
-      onPressedChange={setShowUSDCInputOnTablet}
-      size={ButtonSize.XSmall}
-      shape={ButtonShape.Square}
+    <WithTooltip
+      slotTooltip={
+        <$Tooltip>
+          {!showUSDCInput && usdAmountInput ? (
+            `≈ ${formatNumberOutput(usdAmountInput, OutputType.Fiat, { decimalSeparator, groupSeparator, selectedLocale })}`
+          ) : showUSDCInput && amountInput && id ? (
+            <>
+              ≈ {amountInput} <Tag>{id}</Tag>
+            </>
+          ) : undefined}
+        </$Tooltip>
+      }
+      side="left"
+      align="center"
     >
-      <Icon iconName={IconName.Trade} />
-      {showUSDCInputOnTablet ? 'USD' : id}
-    </$ToggleButton>
+      <$ToggleButton
```

Comment:
couldddd we potentially explore making the icon bigger?

Reply:
<img width="399" alt="image" src="https://github.com/user-attachments/assets/432c3fb3-bfa8-41f7-a332-a58d5b93925e">
maybe ` --button-font: var(--font-small-book);`?